content/js1/sprints/2/prep/index.md

```diff
@@ -8,4 +8,406 @@ backlog= 'Module-JS1'
 backlog_filter= 'Week 2'
 +++
 
+## Learning objectives
 
+- Label and describe the syntactic features of a function declaration
+- Identify a call site
+- Differentiate between a function declaration and a function invocation
+- Propose a function definition to satisfy some problem requirements
+- Given a short program, determine which variable names are _bad_ choices
+- Give examples of when the `undefined` data type may occur in a JavaScript program
+- Construct simple expressions to represent a set of operations
+- Define a _parameter_
+- Identify the return value a call site given its function declaration
+- Identify the value assigned to a parameter when a function is invoked
```

Comment:
I think we could do with a sentence or two explicitly explaining this at the end, and contrasting the terms "parameter" and "argument".

In particular, I don't think we've explained very clearly yet that "a parameter is a variable, which only lives for the scope of the function call, whose value is defined by the caller" - we've hinted at all of these things, but not solidified them all together.